### ISSUE TITLE
Failing tests in PyTest aren't reporting as a failure

### ISSUE THREAD

**Issue by visch** [MeltanoLabs/target-snowflake#90]
See https://github.com/visch/target-snowflake/actions/runs/5545236322/jobs/10124133196 , I was trying some blue green testing with github actions (Expecting Windows jobs to fail, and they are failing in Pytest but it's not being reported that way)

**Comment by edgarrmondragon** [comment]
I see tons of warnings but no Pytest failures. It'd still be a good idea to fail those warnings by running pytest with `-W error`.

**Comment by visch** [comment]
> I see tons of warnings but no Pytest failures. It'd still be a good idea to fail those warnings by running pytest with `-W error`.

You're right! Updated the title of this. Really odd this test actually worked, it means a full end to end test isn't happening right now. Seperate issue incoming!
